assignment/src/com/Question10/UserMainCode.java

```diff
@@ -0,0 +1,36 @@
+package Question10;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class UserMainCode {
+
+    public static List<Integer> performSetOperations(List<Integer> list1, List<Integer> list2, String operationType) {
+        List<Integer> modifiedList = new ArrayList<>();
+        switch (operationType) {
+            case "+":
+                Set<Integer> elimanteCommon = new HashSet<>();
+                elimanteCommon.addAll(list1);
+                elimanteCommon.addAll(list2);
+                modifiedList.addAll(elimanteCommon);
+                break;
+            case "*":
+                for (Integer number : list1) {
+                    if (list2.contains(number)) {
```

Comment:
What if list1 contains duplicate numbers? The modified list may contain those number multiple times.  Pls check.